### PR TITLE
Add responsive navbar with Bootstrap

### PR DESCRIPTION
**Pull request by LuisaFZ**
Add responsive navbar through Bootstrap 4.6.
Made some small changes to texts.
Rename some CSS classes to avoid conflicting with the Bootstrap ones.

### CODE REVIEW

**Comment by santisoler** [comment]
@LuisaFZ I merged `main` into this branch so it has the latest changes and solves some conflicts.
I've also managed to solve some issues: we were using some CCS classes that conflict with the one from Bootstrap, like `container` and `button`.

Now the website looks really nice! :tada:

~Although I wasn't able to make the burger icon to work: I cannot see the menu when I click it. Do you know how to solve this?~

**Comment by santisoler** [comment]
Nevermind the last sentence, we used to have some CSS rules for small displays that used to hide the navbar and resize its height. I just removed them and the burger menu is now working fine.

**Comment by santisoler** [comment]
@aguspesce is working in implementing Bootstrap on the entire website and ditch these custom CSS classes, but that's for another PR. I think this is ready to be merged if you agree.

File: index.html [hunk 1]
```diff
@@ -23,22 +30,57 @@
...
-        <li><a href="#achievements">Achievements</a></li>
-        <li><a href="#events">Coming Events</a></li>
-      </ul>
+    <!-- nav bar -->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="#"
+          ><img
+            class="logo-header"
+            src="images\geolatinas-logo.svg"
```
Comment (santisoler) [inline index.html]:
```suggestion
            src="images/geolatinas-logo.svg"
```

File: index.html [hunk 2]
```diff
@@ -7,8 +7,15 @@
...
     />
 
-    <!-- Main CSS style -->
+    <!-- CSS styles -->
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1"
+      crossorigin="anonymous"
+    />
```
Comment (santisoler) [inline index.html]: Considering that Bootstrap 5 is in beta, shouldn't we choose Bootsrap 4 for now?

**Comment by LuisaFZ** [comment]
No idea, here’s a comparison https://ourtechroom.com/tech/bootstrap4-vs-boostrap5-migrate-differences/

File: index.html (hunk 2 above)
Comment (LuisaFZ) [reply index.html]: I deleted the other comment by mistake, but maybe is best to stick with 4 like you suggest

**Comment by santisoler** [comment]
I think this is ready to go @LuisaFZ ! Congratulations, nice work! 🏅

**Pull request merged**